src/test/java/com/wikia/webdriver/testcases/desktop/discussions/CreatingPostTests.java

```diff
@@ -21,27 +20,30 @@
 
 import org.testng.annotations.Test;
 
-@Execute(onWikia = MobileWikis.DISCUSSIONS_2)
+@Execute(onWikia = "qadiscussions", language = "de")
 @Test(groups = "discussions-creating-posts")
 public class CreatingPostTests extends NewTestTemplate {
 
+  private static final String wikiName = "qadiscussions";
+  private static final String language = "de";
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS00115)